src/main/java/eu/dissco/backend/controller/RestResponseEntityExceptionHandler.java

```diff
@@ -33,7 +33,7 @@ public ResponseEntity<ExceptionResponseWrapper> handleJsonException(JsonProcessi
   @ResponseStatus(HttpStatus.BAD_GATEWAY)
   @ExceptionHandler(IOException.class)
   public ResponseEntity<ExceptionResponseWrapper> handleIOException(java.io.IOException e){
-    logger.error("An IOException has occurred. Check elastic connection.", e);
+    logger.error("An IOException has occurred.", e);
     var exceptionResponse = new ExceptionResponseWrapper(
         HttpStatus.BAD_GATEWAY,
         "ElasticSearch exception",
```

Comment:
Sorry my main comment was about this one

Reply:
fixed